modules/acm_sku/src/ProductManager.php

```diff
@@ -309,9 +309,12 @@ public function synchronizeProducts(array $products = [], $storeId = '') {
         // If skip attribute is set via any event subscriber, skip importing the
         // product.
         if ($product['skip']) {
-          $this->ignoredSkus[] = $product['sku'] . '(SKU doesn\'t meet the criteria for import set for this site.)';
-          $this->ignored++;
-          continue;
+          // We mark the status to disabled so product is deleted if available.
```

Comment:
Comment doesn't match the code.

Reply:
what we do is set the status to disabled so that it is deleted if available and skipped if not available.

there are some cases for image conditions which can fail in next update and product might be added already in system before so we want to delete (and not just skip the current update) the product if available

Reply:
but we should still set ```$this->ignoredSkus[] = $product['sku'] . '(SKU doesn\'t meet the criteria for import set for this site.)';
          $this->ignored++;```, shouldn't we?

Reply:
it might confuse when deleting with two messages (one in ignored and one in deleted)
we already add log here about change of status and later in code it will be either ignored or deleted